src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -78,71 +144,55 @@ html[data-theme="light"] {
     }
   }
 }
+
 /*******************************************************************************
- * color overrides. Here we create some extra --pst-color-* variables and use
- * them to override the value of the corresponding sphinx-design variables.
- * This is easier than re-writing the sphinx-design rules. Even easier would be
- * directly assigning our values to the --sd-color-* variables, but then our
- * downstream users couldn't override *our* colors and have it affect buttons
- * and badges.
- *
- * First, define the extra keys needed to cover the full range of semantic
- * color names used in sphinx-design, then merge them with the names we
- * already define for our own needs.
+ * Dropdowns
  */
-$extra-semantic-colors: (
-  "dark": map-get($pst-semantic-colors, "text-base"),
-  "muted": map-get($pst-semantic-colors, "text-muted"),
-  "light": rgb(201, 201, 201),
-  "black": rgb(0, 0, 0),
-  "white": rgb(255, 255, 255),
-);
+details.sd-dropdown {
+  // We apply a border in the header and content to allow for color
+  border-left: 0px !important;
 
-$all-colors: map-merge($pst-semantic-colors, $extra-semantic-colors);
+  // Drop shadow should behave same as admonitions
+  @include box-shadow();
+  summary.sd-card-header {
+    position: relative; // So background color works
+    font-weight: 600;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+    border-left: 1px solid var(--sd-color-card-border);
```

Comment:
just wondering how these were chosen (and whether it makes sense to have CSS variables for them?)...  anytime we're writing style rules to "make this component look like this other component" it makes me wonder

Reply:
I think in this case I am just literally copying the border rule that sphinx design uses. The problem was that we are putting the border on the title and content blocks, while sphinx design was putting it on the whole details block only when it was closed, and putting it on the title only when it was open, or something like that.

Maybe I should add a comment to explain that?